numpy/core/src/multiarray/ctors.c

```diff
@@ -1468,12 +1468,25 @@ PyArray_FromAny(PyObject *op, PyArray_Descr *newtype, int min_depth,
     }
 
     /* Create a new array and copy the data */
-    ret = (PyArrayObject *)PyArray_NewFromDescr(
+    ret = (PyArrayObject *)PyArray_NewFromDescr_int(
             &PyArray_Type, dtype, ndim, dims, NULL, NULL,
-            flags&NPY_ARRAY_F_CONTIGUOUS, NULL);
+            flags&NPY_ARRAY_F_CONTIGUOUS, NULL, NULL, 0, 1);
```

Comment:
As a refresher, what is this `1` argument?

Reply:
```
PyArray_NewFromDescr_int(
        PyTypeObject *subtype, PyArray_Descr *descr, int nd,
        npy_intp const *dims, npy_intp const *strides, void *data,
        int flags, PyObject *obj, PyObject *base, int zeroed,
        int allow_emptystring)
```